src/Components/Header/Header.js

```diff
@@ -5,23 +5,21 @@ import './Header.css';
 import { NavLink } from 'react-router-dom';
 
 const Header = ({ togView, currentUser }) => {
-  const [showNotifs, setShowNotifs] = useState(false);
+  const [showNotifs, setShowNotifs] = useState(false)
 
   const seeNotifs = (event) => {
     event.preventDefault()
     if (!showNotifs)  {
-      console.log("showing notifications")
       setShowNotifs(true)
   } else {
     setShowNotifs(false)
-    console.log("hiding notifications")
   }
 }
 
   return (
     <div className="header-container">
       <div className="dropdown">
-          <p className={`${showNotifs === false ? "dropdown-show" : "dropdown-hide"}`}>notification text will go here</p>
+          <p className={`${showNotifs === false ? "dropdown-hide" : "dropdown-show"}`}>notification text will go here</p>
```

Comment:
I feel like based upon how patrick has set up the WebSockets we might also be able to emit a message to all users upon the click of that button, and just have it display within the notification tab, might be weird to work through though tbh

Reply:
I like the idea of having the notification pop up here in the header.